tests/python/test_tutorial.py

```diff
@@ -0,0 +1,277 @@
+import pytest
```

Comment:
these tests are meant to be for the python sdk, not just for the tutorial. so it does not make sense to name it based on the tutorial (btw, its also confusing to have tests/test_tutorial.py and tests/python/test_tutorial.py). So I think we should to the following:

- clients/python/tests/ - all the test files will be here
- clients/python/README - explains how where the python is generated and how to run the python client tests.

Reply:
Would it make sense to put these in `clients/`? It is not a directory in this repo right now and all of the clients are actually generated in `target/clients/` which is gitignored.

Reply:
i actually want to create this directory, because otherwise, its actually very confusing to find where the clients are. unless someone really knows where to look in build.sbt, i think its hard to understand what is going on. so i actually want to make it obvious by having a top-level directory called clients, and then slowly start using them for stuff. so lets start that right now :D 

maybe we should generate the code in clients/python/target/ ? is that doable in this PR?

Reply:
Moved tests and changed client generation to `clients/python`!

Reply:
I also separated the tests out into separate files based on the part of the API